client/components/theme-type-badge/index.tsx

```diff
@@ -44,6 +46,8 @@ const ThemeTypeBadge = ( {
 		} );
 	}, [ type, themeId, isLockedStyleVariation ] );
 
+	const themeSoftware = themeSoftwareSet[ 0 ];
```

Comment:
In some places, we are hard coding the `themeSoftwareSet[ 0 ]` to get the first one. Do we plan to have multiple plugins bundled with a team? Sensei LMS + WooCommerce, for example.

Reply:
For this thread, it may be worth considering a way to identify the _primary_ software set: what does the user think they are getting/installing when they pick this theme?

Reply:
> I think the question is more 'do we plan to have multiple software sets for a theme'.

Sorry! That's correct!

> I think the question is more 'do we plan to have multiple software sets for a theme'.

Could you elaborate a little more your idea @daledupreez? I'm not sure if I understand it.
Do you mean improving the UI with the details of what would be installed? Or something in the showcase editor to set the primary software set? Or something else not related to this at all? 🤭